2022-07-12 16:44:07 error: [GET] /api/v1/auth/login >> StatusCode:: 500, Message:: Cannot read properties of undefined (reading 'authService')
2022-07-12 16:44:44 error: [GET] /api/v1/auth/login >> StatusCode:: 401, Message:: Invalid credentials
2022-07-12 16:45:15 error: [GET] /api/v1/auth/login >> StatusCode:: 401, Message:: Invalid credentials
2022-07-12 16:45:21 error: [GET] /api/v1/auth/login >> StatusCode:: 401, Message:: Invalid credentials
2022-07-12 16:45:24 error: [GET] /api/v1/auth/login >> StatusCode:: 500, Message:: data and hash arguments required
2022-07-12 16:45:30 error: [GET] /api/v1/auth/login >> StatusCode:: 401, Message:: Invalid credentials
2022-07-12 17:21:44 error: uncaughtException: Entity 'Profile' was not discovered, please make sure to provide it in 'entities' array when initializing the ORM (used in User.profile)
MetadataError: Entity 'Profile' was not discovered, please make sure to provide it in 'entities' array when initializing the ORM (used in User.profile)
    at Function.fromUnknownEntity (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/errors.js:148:16)
    at /Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataValidator.js:55:46
    at Array.forEach (<anonymous>)
    at /Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataValidator.js:53:67
    at Array.forEach (<anonymous>)
    at MetadataValidator.validateDiscovered (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataValidator.js:53:20)
    at MetadataDiscovery.findEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:92:24)
    at MetadataDiscovery.discover (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:34:9)
    at MikroORM.discoverEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:90:25)
    at Function.init (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:45:9)
2022-07-12 17:22:03 error: uncaughtException: Entity 'Profile' was not discovered, please make sure to provide it in 'entities' array when initializing the ORM (used in User.profile)
MetadataError: Entity 'Profile' was not discovered, please make sure to provide it in 'entities' array when initializing the ORM (used in User.profile)
    at Function.fromUnknownEntity (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/errors.js:148:16)
    at /Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataValidator.js:55:46
    at Array.forEach (<anonymous>)
    at /Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataValidator.js:53:67
    at Array.forEach (<anonymous>)
    at MetadataValidator.validateDiscovered (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataValidator.js:53:20)
    at MetadataDiscovery.findEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:92:24)
    at MetadataDiscovery.discover (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:34:9)
    at MikroORM.discoverEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:90:25)
    at Function.init (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:45:9)
2022-07-12 17:22:33 error: uncaughtException: Please provide either 'type' or 'entity' attribute in Profile.user
Error: Please provide either 'type' or 'entity' attribute in Profile.user
    at ReflectMetadataProvider.initPropertyType (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/ReflectMetadataProvider.js:14:19)
    at /Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/ReflectMetadataProvider.js:9:54
    at ReflectMetadataProvider.initProperties (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataProvider.js:34:23)
    at ReflectMetadataProvider.loadEntityMetadata (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/ReflectMetadataProvider.js:9:9)
    at MetadataDiscovery.discoverEntity (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:224:13)
    at MetadataDiscovery.discoverReferences (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:163:13)
    at MetadataDiscovery.findEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:90:9)
    at MetadataDiscovery.discover (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:34:9)
    at MikroORM.discoverEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:90:25)
    at Function.init (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:45:9)
2022-07-12 17:23:10 error: uncaughtException: Please provide either 'type' or 'entity' attribute in Profile.user
Error: Please provide either 'type' or 'entity' attribute in Profile.user
    at ReflectMetadataProvider.initPropertyType (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/ReflectMetadataProvider.js:14:19)
    at /Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/ReflectMetadataProvider.js:9:54
    at ReflectMetadataProvider.initProperties (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataProvider.js:34:23)
    at ReflectMetadataProvider.loadEntityMetadata (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/ReflectMetadataProvider.js:9:9)
    at MetadataDiscovery.discoverEntity (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:224:13)
    at MetadataDiscovery.discoverReferences (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:163:13)
    at MetadataDiscovery.findEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:90:9)
    at MetadataDiscovery.discover (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:34:9)
    at MikroORM.discoverEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:90:25)
    at Function.init (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:45:9)
2022-07-12 17:23:17 error: uncaughtException: Please provide either 'type' or 'entity' attribute in Profile.user
Error: Please provide either 'type' or 'entity' attribute in Profile.user
    at ReflectMetadataProvider.initPropertyType (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/ReflectMetadataProvider.js:14:19)
    at /Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/ReflectMetadataProvider.js:9:54
    at ReflectMetadataProvider.initProperties (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataProvider.js:34:23)
    at ReflectMetadataProvider.loadEntityMetadata (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/ReflectMetadataProvider.js:9:9)
    at MetadataDiscovery.discoverEntity (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:224:13)
    at MetadataDiscovery.discoverReferences (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:163:13)
    at MetadataDiscovery.findEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:90:9)
    at MetadataDiscovery.discover (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:34:9)
    at MikroORM.discoverEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:90:25)
    at Function.init (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:45:9)
2022-07-12 17:23:29 error: uncaughtException: Profile.user has unknown 'mappedBy' reference: User.user
MetadataError: Profile.user has unknown 'mappedBy' reference: User.user
    at Function.fromMessage (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/errors.js:172:16)
    at Function.fromWrongReference (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/errors.js:117:30)
    at MetadataValidator.validateInverseSide (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataValidator.js:97:42)
    at MetadataValidator.validateBidirectional (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataValidator.js:76:18)
    at MetadataValidator.validateEntityDefinition (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataValidator.js:33:18)
    at MetadataDiscovery.processEntity (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:354:24)
    at MetadataDiscovery.processDiscoveredEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:76:43)
    at MetadataDiscovery.discover (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/metadata/MetadataDiscovery.js:35:9)
    at MikroORM.discoverEntities (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:90:25)
    at Function.init (/Users/gagetylee/callio-api/node_modules/@mikro-orm/core/MikroORM.js:45:9)
2022-07-12 18:07:23 error: [POST] /api/v1/user >> StatusCode:: 409, Message:: Email is already in use
2022-07-12 19:21:38 error: uncaughtException: write EPIPE
Error: write EPIPE
    at process.target._send (node:internal/child_process:844:20)
    at process.target.send (node:internal/child_process:717:19)
    at Object.exports.send (/Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/ipc.js:17:14)
    at /Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/wrap.js:88:9
    at Object.nodeDevHook [as .js] (/Users/gagetylee/callio-api/node_modules/ts-node-dev/lib/hook.js:61:17)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/gagetylee/callio-api/node_modules/swagger-jsdoc/src/lib.js:1:19)
